.travis.yml

```diff
@@ -1,10 +1,12 @@
+dist: xenial
```

Comment:
Why Xenial? What for?

Reply:
It is newer, i.e. upgraded packages etc.
And it provides `3.7` out of the box, instead of 3.7-dev.

Reply:
Ah! I get it now. I was assuming Travis was using a newer version.